protobuf/policy.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+ 
+package policy;
```

Comment:
Suggest to provide status, policy and hostPolicy with starting Caps. Looks better. 
Status, Policy and HostPolicy

Reply:
![image](https://user-images.githubusercontent.com/47106543/146413801-2c09f2ab-f8af-44f7-badb-24cd534f094e.png)
This changes the json field names to Title Case which doesn't seem the standard convention. What do you think?